Enyim.Caching/Configuration/ConfigurationHelper.cs

```diff
@@ -98,9 +98,8 @@ public static EndPoint ResolveToEndPoint(string host, int port)
             // parse as an IP address
             if (!IPAddress.TryParse(host, out address))
             {
-                //Call System.Net.Dns.GetHostEntry(string hostNameOrAddress) that implements for netstandard 2.0
-                var method = typeof(System.Net.Dns).GetMethod("GetHostEntry", BindingFlags.Public | BindingFlags.Static);
-                var addresses = ((IPHostEntry)method.Invoke(null, new object[] { host })).AddressList;
+                var hostEntry = Dns.GetHostEntryAsync(host).Result;
```

Comment:
It is not a apart of netstandard 1.3 and  netstandard 1.5 apis. It is  a apart of netstandard 2.0 apis. See dotnet/corefx@0539e53
The reflection is a workrouand for netstandard 1.5 that EnyimMemcachedCore works with.